release/kata-deploy-binaries.sh

```diff
@@ -167,6 +167,18 @@ install_firecracker() {
 	popd
 }
 
+# Install static cloud-hypervisor asset
+install_cloud_hypervisor() {
+	info "build static cloud-hypervisor"
+	"${script_dir}/../static-build/cloud-hypervisor/build-static-clh.sh"
+	info "Install static cloud-hypervisor"
+	mkdir -p "${destdir}/opt/kata/bin/"
+	sudo install -D --owner root --group root --mode 0744 cloud-hypervisor/cloud-hypervisor "${destdir}/opt/kata/bin/cloud-hypervisor"
+	pushd "${destdir}"
+	tar -czvf ../kata-cloud-hypervisor-static.tar.gz *
+	popd
+}
```

Comment:
@jcvenegas This needs to be updated as well to include the function name:
https://github.com/kata-containers/packaging/blob/030dfd92ff62270c2c9e73378973454ace11251b/artifact-list.sh